interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -16,6 +16,9 @@
 
 package io.grpc.testing.integration;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
This isn't a test file, so don't use asserts. Instead you could use things like IllegalArgumentException or IllegalStateException.

Reply:
removed per below comment